dask/widgets/__init__.py

```diff
@@ -9,11 +9,11 @@
         '  python -m pip install "dask[diagnostics]" --upgrade  # or python -m pip install'
     )
     exception = e  # Explicit reference for e as it will be lost outside the try block
-    FILTERS = dict()
+    FILTERS = {}
     TEMPLATE_PATHS = []
 
-    def get_environment(*args, **kwargs):
+    def get_environment(*args, **kwargs):  # type: ignore
```

Comment:
Doesn't seem like it costs anything to make the signatures of this match that of `widgets.get_{environment, template}`:
```suggestion
    def get_environment() -> Environment:
```
Is anyone using this and passing args/kwargs into the function?

Reply:
fixed